terraform/main.tf

```diff
@@ -0,0 +1,169 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+    }
+  }
+}
+
+# Configure the AWS Provider
+provider "aws" {
+  region = "us-east-2"
+  access_key = aws_access_key
```

Comment:
Can these be environment variables rather than tfvars?

Reply:
Done